### PR TITLE
Corrected documentation

### LINKED ISSUE #31013: Incorrect Documentation: Configuring Resharper
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/.github/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/copilot/configuring-github-copilot/configuring-github-copilot-in-your-environment

### What part(s) of the article would you like to see updated?

Step 1 under Configuring ReSharper for {% data variables.product.prodname_copilot %} is incorrect. Step 2 and forward are not available from the Visual Studio options menu. What the author intended to say in step 1 was Extensions => Reshaper => Options. 

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by C0derPr0**
### Why:

Step 1 for the section "Configuring ReSharper for GitHub Copilot" and the associated image was incorrect.

Closes: [31013](https://github.com/github/docs/issues/31013)

### What's being changed (if available, include any code snippets, screenshots, or gifs):

I changed the incorrect content under the section "Configuring Resharper for GitHub Copilot."

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@C0derPr0 Thanks so much for opening a PR! I'll get this triaged for review ✨

File: content/copilot/configuring-github-copilot/configuring-github-copilot-in-your-environment.md [hunk 1]
```diff
@@ -233,8 +233,8 @@ If you don't want to use the default keyboard shortcuts in {% data variables.pro
 
 If you use ReSharper, {% data variables.product.prodname_copilot %} may work best when you configure ReSharper to use {% data variables.product.prodname_copilot %}'s native IntelliSense. For more information about ReSharper, see the [ReSharper documentation](https://www.jetbrains.com/resharper/documentation/documentation.html)
 
-1. In the {% data variables.product.prodname_vs %} menu bar, under **Tools**, click **Options**.
-   ![Screenshot of {% data variables.product.prodname_vs %} menu bar. In the "Tools" menu, the "Options" item is highlighted with an orange outline.](/assets/images/help/copilot/vs-toolbar-options.png)
+1. In the Resharper menu bar, under **Extensions**, then **Resharper**, click **Options**.
```
Comment (hubwriter) [inline content/copilot/configuring-github-copilot/configuring-github-copilot-in-your-environment.md:236]: It's the Visual Studio menu bar:
```suggestion
1. In the {% data variables.product.prodname_vs %} menu bar, under **Extensions**, click **ReSharper**, then click **Options**.
```

File: content/copilot/configuring-github-copilot/configuring-github-copilot-in-your-environment.md [hunk 2]
```diff
@@ -233,8 +233,8 @@ If you don't want to use the default keyboard shortcuts in {% data variables.pro
...
+   ![Screenshot of Resharper menu bar. In the "Resharper" menu, the "Options" item is highlighted with an orange outline.](/assets/images/help/copilot/resharper-toolbar-options.png)
```
Comment (hubwriter) [inline content/copilot/configuring-github-copilot/configuring-github-copilot-in-your-environment.md:237]: The screenshot doesn't match the style of our screenshots. However, I don't think we need a screenshot here. We should just delete this line.

```suggestion
```

**Review by hubwriter — CHANGES_REQUESTED** [review]
@C0derPr0 - Many thanks for raising the docs issue, and this PR. 🎖️ 

I've made a couple of little suggestions which you can click to commit.

Thanks for including a screenshot but I don't think it's necessary here, and we've recently been trying to cut down on unnecessary screenshots in the docs. Could you delete the screenshot file from this PR? Thanks

**Comment by C0derPr0** [comment]
Sorry to be a bother..., but it says, "Merging is blocked until requested changes are addressed." I committed the requested changes, so I don't know why it is blocked.

**Comment by nguyenalex836** [comment]
@C0derPr0 Thanks for letting us know! @hubwriter may need to re-review after the suggestions have been committed, and I've just requested a re-review from him 💛

**Comment by hubwriter** [comment]
Hi @C0derPr0 - thanks for making those changes.

Can you delete [resharper-toolbar-options.png](https://github.com/github/docs/pull/31025/files#diff-791530c19159e3de4546e02cf363fe585e304d7d1111cf5b6f6c7b50307a1339) from your branch? Then I'll get this merged.

**Comment by C0derPr0** [comment]
@hubwriter Okay, I deleted the image. Sorry about that, I committed your requested change several days ago and thought that included deleting the image.

**Review by hubwriter — APPROVED** [review]
@C0derPr0 - Great. Thanks for the quick response.

I'll get this merged now. Many thanks for helping with the docs. Much appreciated. 👍

**Comment by hubwriter** [comment]
Now published:
https://docs.github.com/en/copilot/configuring-github-copilot/configuring-github-copilot-in-your-environment#configuring-resharper-for-github-copilot

**Pull request merged**